src/Illuminate/Notifications/Channels/BroadcastChannel.php

```diff
@@ -40,7 +41,9 @@ public function send($notifiable, Notification $notification)
         $message = $this->getData($notifiable, $notification);
 
         $event = new BroadcastNotificationCreated(
-            $notifiable, $notification, is_array($message) ? $message : $message->data
```

Comment:
I would remove this formatting change. not really applicable to your suggestion.

Reply:
Nah, you are right. It was done automatically, I'll remove it :-)